src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/config/HmppsAssessForEarlyReleaseApiExceptionHandler.kt

```diff
@@ -29,6 +30,17 @@ class HmppsAssessForEarlyReleaseApiExceptionHandler {
       ),
     ).also { log.info("Validation exception: {}", e.message) }
 
+  @ExceptionHandler(HttpMessageNotReadableException::class)
+  fun handleHttpMessageNotReadableException(e: ValidationException): ResponseEntity<ErrorResponse> = ResponseEntity
+    .status(BAD_REQUEST)
+    .body(
+      ErrorResponse(
+        status = BAD_REQUEST,
+        userMessage = "Validation failure: ${e.message}",
+        developerMessage = e.message,
+      ),
+    ).also { log.info("Validation exception: {}", e.message) }
+
```

Comment:
Not sure if I should change it to something other than a validation error, but to me in kind of makes sense as it's a JSR validation annotation

Reply:
is this when enum values are incorrect?

Reply:
HttpMessageNotReadableException is thrown when some of these fail:
```
  @field:NotNull
  @field:NotEmpty
```
Seems a little strange really as I expected a ValidationException